lib/tracing-policy.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * Copyright 2015 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+function RateLimiterPolicy(samplesPerSecond) {
+  var that = this;
+  var timeout = 1000 / samplesPerSecond;
+  this.enabled = false;
+  timeout = timeout < 1 ? 1 : timeout;
```

Comment:
We should make it more explicit in config.js that we definitely do not support samplingRates faster than 1000/sec.